docs/src/tool/index.md

```diff
@@ -3,4 +3,363 @@
 This document describes the `reach` tool: how to install it and how to use it.
 
 + @{seclink("ref-install")}
-+ @{seclink("ref-usage")}
\ No newline at end of file
++ @{seclink("ref-usage")}
```

Comment:
I don't think these look good anymore and can just be removed

Reply:
I thought about that, but then thought it might be handy to have for the mobile version. I was looking at it on my phone and there is no way to access the left-side navigation, and scrolling down can be a bit tedious. I can remove them, though. It might be interesting to check web stats every so often to see how people are accessing the documentation for insight into design. What are your thoughts?

Reply:
I trust your judgment. However, you can get the navigation... there should be hamburger at the top of the page that you can click on and get